site/src/components/badge.md

```diff
@@ -22,12 +22,13 @@ Badges use short text, color, and an optional icon for quick recognition.
 
 ### Sizes
 
-The following classes are available for resizing: `badge-sm`, `badge-lg`.
```

Comment:
GH wouldn't let me do a suggestion for this, but I think we should remove the old Figma URL. Just delete it for now.
```
figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fdesign%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fm%3Dauto%26node-id%3D3381-10722%26t%3DZhpJOvxIFDBwfkR7-1

Reply:
removed